app/src/main/res/layout/fragment_input_hostname.xml

```diff
@@ -1,63 +1,79 @@
 <?xml version="1.0" encoding="utf-8"?>
 <FrameLayout xmlns:android="http://schemas.android.com/apk/res/android"
-        xmlns:app="http://schemas.android.com/apk/res-auto"
-        xmlns:tools="http://schemas.android.com/tools"
-        android:layout_width="match_parent"
-        android:layout_height="match_parent"
-        android:background="?attr/colorPrimaryDark">
+    xmlns:app="http://schemas.android.com/apk/res-auto"
+    xmlns:tools="http://schemas.android.com/tools"
+    android:layout_width="match_parent"
+    android:layout_height="match_parent"
+    android:background="?attr/colorPrimaryDark"
+    tools:context="chat.rocket.android.fragment.add_server.InputHostnameFragment">
 
     <LinearLayout
-            android:layout_width="wrap_content"
-            android:layout_height="wrap_content"
-            android:layout_gravity="center"
-            android:background="@color/white"
-            android:minWidth="288dp"
-            android:orientation="horizontal"
-            android:padding="@dimen/margin_24">
+        android:layout_width="wrap_content"
+        android:layout_height="wrap_content"
+        android:layout_gravity="center"
+        android:minWidth="288dp"
+        android:padding="@dimen/margin_24"
+        android:background="@color/white"
+        android:orientation="vertical">
 
         <LinearLayout
+            android:layout_width="match_parent"
+            android:layout_height="match_parent"
+            android:orientation="horizontal">
+
+            <LinearLayout
                 android:layout_width="0px"
                 android:layout_height="wrap_content"
                 android:layout_weight="1"
                 android:orientation="vertical">
 
-            <TextView
+                <TextView
                     android:layout_width="match_parent"
                     android:layout_height="wrap_content"
+                    android:paddingStart="4dp"
```

Comment:
It is a workaround to align the TextView w/ the EditText. Take a look:

Before
![before](https://user-images.githubusercontent.com/11776825/27844848-cb9c1464-60fd-11e7-8e57-64c9a1be482e.png)

After:
![after](https://user-images.githubusercontent.com/11776825/27844847-cb9955e4-60fd-11e7-9488-302f50a64934.png)